_data/navigation.yml

```diff
@@ -0,0 +1,24 @@
+---
+- title: 'Work'
+  link: '/#work'
+  type: local
+
+- title: 'About'
+  link: '/#about'
+  type: local
+
+- title: 'Join VAC'
+  link: '/#join'
+  type: local
+
+- title: 'Research log'
+  link: '/research-log'
+  type: local
+
+- title: 'Specs'
+  link: 'https://specs.vac.dev'
```

Comment:
This should be rfc.vac.dev, it is live there

Reply:
Changed